[2023-02-20 00:46:39,691] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: autoetl-coordinates.get_input_data manual__2023-02-20T00:46:37.009572+00:00 [queued]>
[2023-02-20 00:46:39,705] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: autoetl-coordinates.get_input_data manual__2023-02-20T00:46:37.009572+00:00 [queued]>
[2023-02-20 00:46:39,706] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-20 00:46:39,707] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-20 00:46:39,708] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-20 00:46:39,726] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_input_data> on 2023-02-20 00:46:37.009572+00:00
[2023-02-20 00:46:39,737] {standard_task_runner.py:52} INFO - Started process 4724 to run task
[2023-02-20 00:46:39,742] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'autoetl-coordinates', 'get_input_data', 'manual__2023-02-20T00:46:37.009572+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/autoetl-coordinates.py', '--cfg-path', '/tmp/tmpz1f2nz99', '--error-file', '/tmp/tmp_n3ipnj1']
[2023-02-20 00:46:39,745] {standard_task_runner.py:80} INFO - Job 17: Subtask get_input_data
[2023-02-20 00:46:39,815] {task_command.py:369} INFO - Running <TaskInstance: autoetl-coordinates.get_input_data manual__2023-02-20T00:46:37.009572+00:00 [running]> on host 83ee186d463f
[2023-02-20 00:46:39,896] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=autoetl-coordinates
AIRFLOW_CTX_TASK_ID=get_input_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-20T00:46:37.009572+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-20T00:46:37.009572+00:00
[2023-02-20 00:46:39,898] {logging_mixin.py:115} INFO - Get Locations URL and Load into Pandas Dataframe
[2023-02-20 00:46:43,648] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:189 DtypeWarning: Columns (11,12,14,20) have mixed types.Specify dtype option on import or set low_memory=False.
[2023-02-20 00:46:56,508] {logging_mixin.py:115} INFO - Get buildings data
[2023-02-20 00:47:17,737] {logging_mixin.py:115} INFO - Get parcels data
[2023-02-20 00:47:18,370] {logging_mixin.py:115} WARNING - Downloading...
[2023-02-20 00:47:18,372] {logging_mixin.py:115} WARNING - From: https://drive.google.com/uc?id=137bCzf0qEPLwKTgNxeAAjTs5MvyTlxWA
[2023-02-20 00:47:18,373] {logging_mixin.py:115} WARNING - To: /opt/***/data/inputs/ms_hinds_parcels.ndgeojson.gz
[2023-02-20 00:47:18,384] {logging_mixin.py:115} WARNING -   0%|          | 0.00/42.1M [00:00<?, ?B/s]
[2023-02-20 00:47:18,505] {logging_mixin.py:115} WARNING -   7%|7         | 3.15M/42.1M [00:00<00:01, 26.4MB/s]
[2023-02-20 00:47:18,607] {logging_mixin.py:115} WARNING -  16%|#6        | 6.82M/42.1M [00:00<00:01, 31.6MB/s]
[2023-02-20 00:47:18,762] {logging_mixin.py:115} WARNING -  25%|##4       | 10.5M/42.1M [00:00<00:01, 27.3MB/s]
[2023-02-20 00:47:18,863] {logging_mixin.py:115} WARNING -  34%|###3      | 14.2M/42.1M [00:00<00:00, 30.4MB/s]
[2023-02-20 00:47:18,975] {logging_mixin.py:115} WARNING -  42%|####2     | 17.8M/42.1M [00:00<00:00, 31.2MB/s]
[2023-02-20 00:47:19,090] {logging_mixin.py:115} WARNING -  51%|#####1    | 21.5M/42.1M [00:00<00:00, 31.4MB/s]
[2023-02-20 00:47:19,204] {logging_mixin.py:115} WARNING -  61%|######1   | 25.7M/42.1M [00:00<00:00, 33.2MB/s]
[2023-02-20 00:47:19,307] {logging_mixin.py:115} WARNING -  70%|######9   | 29.4M/42.1M [00:00<00:00, 33.8MB/s]
[2023-02-20 00:47:19,409] {logging_mixin.py:115} WARNING -  81%|########1 | 34.1M/42.1M [00:01<00:00, 37.5MB/s]
[2023-02-20 00:47:19,512] {logging_mixin.py:115} WARNING -  96%|#########5| 40.4M/42.1M [00:01<00:00, 44.6MB/s]
[2023-02-20 00:47:19,546] {logging_mixin.py:115} WARNING - 100%|##########| 42.1M/42.1M [00:01<00:00, 36.2MB/s]
[2023-02-20 00:47:19,564] {logging_mixin.py:115} INFO - Extract and save gzip file
[2023-02-20 00:47:32,670] {python.py:173} INFO - Done. Returned value was: None
[2023-02-20 00:47:32,731] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=autoetl-coordinates, task_id=get_input_data, execution_date=20230220T004637, start_date=20230220T004639, end_date=20230220T004732
[2023-02-20 00:47:32,854] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-20 00:47:32,961] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
